Tests/NIOCoreTests/TimeAmountTests.swift

```diff
@@ -43,4 +43,52 @@ class TimeAmountTests: XCTestCase {
         lhs -= rhs
         XCTAssertEqual(lhs, .nanoseconds(0))
     }
+
+#if swift(>=5.7)
```

Comment:
The swift check has to live inside the function, since otherwise old Swift version don't compile.

Reply:
Is that because of the unguarded addition to `TimeAmountTests+XCTest.swift`. I'm happy to move the guard in the function but wanted to check if it would instead be cleaner to instead add another guard in the `+XCTest.swift` to mirror this one. That way the test won't appear at all on older versions, cf. being an empty test. WDYT?

Reply:
> Is that because of the unguarded addition to `TimeAmountTests+XCTest.swift`. 

Exactly.

> I'm happy to move the guard in the function but wanted to check if it would instead be cleaner to instead add another guard in the `+XCTest.swift` to mirror this one. That way the test won't appear at all on older versions, cf. being an empty test. WDYT?

I don't think it matters all that much which way it's done. I suspect updating the script to do this is probably more hassle than it's worth though.